gbpservice/nfp/orchestrator/drivers/orchestration_driver.py

```diff
@@ -634,6 +640,10 @@ def create_network_function_device(self, device_data,
                                           by=len(interfaces))
             return None
 
+        vendor_data = vendor_data_result.get('result', None)
```

Comment:
What does vendor_data_results contains ? From where 'result' key is getting added in vendor_data_result ?
If key is always present, then why do we need to do  vendor_data_result.get('result', None), we can perform vendor_data_result['result'], 
Also by default get returns None, no need to explicitly specify None.